jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -594,12 +596,13 @@ static URL codeSource(Class<?> callerClass) {
  * @param		newIn 		the new value for in.
  */
 public static void setIn(InputStream newIn) {
+	/*[IF JAVA_SPEC_VERSION < 24]*/
 	@SuppressWarnings("removal")
 	SecurityManager security = System.getSecurityManager();
 	if (security != null)	{
```

Comment:
While you're here, please replace the tab after `)` with a space. Similar changes would be welcome, e.g. on lines 619, 636 and 2006, while the missing braces should be added near lines 924, 948, 1014, 1057, 1284, 1550, 1565.